frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -242,44 +243,13 @@ const composeAndPushNotificationMessages = (
 ) => {
     const username = formData.value.userId.toUpperCase();
     if (successIdList.length > 0) {
-        pushNotification(
-            Severity.success,
-            `${username} ${
-                successIdList[0] === ''
-                    ? `was successfully added with the role ${
-                          getSelectedRole()?.role_name
-                      }`
-                    : `was successfully added with Client IDs: ${successIdList.join(
-                          ', '
-                      )}`
-            }`
-        );
+        setNotificationMsg(successIdList, username, Severity.success);
     }
     if (warningIdList.length > 0) {
-        pushNotification(
-            Severity.warning,
-            `${username} ${
-                warningIdList[0] === ''
-                    ? `already exists with the role ${
-                          getSelectedRole()?.role_name
-                      }`
-                    : `already exists with Client IDs: ${warningIdList.join(
-                          ', '
-                      )}`
-            }`
-        );
+        setNotificationMsg(warningIdList, username, Severity.warning );
```

Comment:
let's remove the space between "warning" and ")"

Reply:
sure, will do!